tools/db_stress.cc

```diff
@@ -1807,8 +1851,31 @@ class StressTest {
         l.reset(new MutexLock(
             shared->GetMutexForKey(rand_column_family, rand_key)));
       }
+
       auto column_family = column_families_[rand_column_family];
 
+      if (FLAGS_acquire_snapshot_one_in > 0 &&
+          thread->rand.Uniform(FLAGS_acquire_snapshot_one_in) == 0) {
+        auto snapshot = db_->GetSnapshot();
+        ReadOptions ropt;
+        ropt.snapshot = snapshot;
+        std::string value_at;
+        auto status_at = db_->Get(ropt, column_family, key, &value_at);
```

Comment:
do you mind adding a comment about what we're verifying? it's not obvious why we're verifying a single key that might not have been overwritten.